File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/hyperlinks/GetHyperlinkWorksheet.java
package com.aspose.cells.cloud.examples.hyperlinks;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Path;

public class GetHyperlinkWorksheet {

    public static void main(String... args) throws IOException {
        // ExStart: get-hyperlink-worksheet
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            String input = "sample1.xlsx";
            Path inputFile = Utils.getPath(GetHyperlinkWorksheet.class, input);
            String sheetName = "Sheet1";
            Integer linkIndex = 0;

            storageApi.PutCreate(input, null, Utils.STORAGE, inputFile.toFile());

            com.aspose.cells.model.HyperlinkResponse apiResponse = cellsApi.GetWorkSheetHyperlink(input, sheetName,
                    linkIndex, Utils.STORAGE, Utils.FOLDER);

            System.out.println(" Link : " + apiResponse.getHyperlink().getAddress());
        }

        catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: get-hyperlink-worksheet

    }

}

File: Examples/Perl/hyperlinks/GetHyperlinkWorksheet.pl
#ExStart:1
use lib 'lib';
use strict;
use warnings;
use utf8;
use File::Slurp; # From CPAN
use JSON;

use AsposeStorageCloud::StorageApi;
use AsposeStorageCloud::ApiClient;
use AsposeStorageCloud::Configuration;
use AsposeCellsCloud::CellsApi;
use AsposeCellsCloud::ApiClient;
use AsposeCellsCloud::Configuration;

my $configFile = '../Config/config.json';
my $configPropsText = read_file($configFile);
my $configProps = decode_json($configPropsText);
my $data_path = '../../../Data/';
my $out_path = $configProps->{'out_folder'};

$AsposeCellsCloud::Configuration::app_sid = $configProps->{'app_sid'};
$AsposeCellsCloud::Configuration::api_key = $configProps->{'api_key'};
$AsposeCellsCloud::Configuration::debug = 1;
$AsposeStorageCloud::Configuration::app_sid = $configProps->{'app_sid'};
$AsposeStorageCloud::Configuration::api_key = $configProps->{'api_key'};

# Instantiate Aspose.Storage and Aspose.Cells API SDK
my $storageApi = AsposeStorageCloud::StorageApi->new();
my $cellsApi = AsposeCellsCloud::CellsApi->new();

# Set input file name
my $name = 'Sample_Test_Book.xls';
my $sheetName = 'Sheet2';	
my $hyperlinkIndex = 0;

# Upload file to aspose cloud storage 
my $response = $storageApi->PutCreate(Path => $name, file => $data_path.$name);
# Invoke Aspose.Cells Cloud SDK API to get a specific hyperlink from a worksheet                             
$response = $cellsApi->GetWorkSheetHyperlink(name=> $name, sheetName=>$sheetName, hyperlinkIndex=>$hyperlinkIndex );

if($response->{'Status'} eq 'OK'){
	my $hyperlink = $response->{'Hyperlink'};
    print "\n Hyperlink Address " . " : " . $hyperlink->{'Address'};
}
#ExEnd:1